src/main/java/com/google/firebase/messaging/WebpushFcmOptions.java

```diff
@@ -0,0 +1,23 @@
+package com.google.firebase.messaging;
+
+import com.google.api.client.util.Key;
+
+/**
+ * Represents options for features provided by the FCM SDK for Web.
+ * Can be included in {@link WebpushConfig}. Instances of this class are thread-safe and immutable.
+ */
+public class WebpushFcmOptions {
```

Comment:
This needs to have:
1. A private no-arg constructor
2. A Builder API
3. A `public static WebpushFcmOptions withLink(String link)` helper method.